api_sentry.py

```diff
@@ -7,13 +7,16 @@
 import os
 import sys
 
+from datetime import datetime
```

Comment:
Instead of importing this here, could we use any of the functions or add one in the datetime_utils.py module?